resources/js/pages/library/view.vue

```diff
@@ -64,6 +64,7 @@
                 :condition="conditions[result.condition].name"
                 :transaction-type="transactionType[result.type].nameForCustomer"
                 :distance="result.bookshelf.distance"
+                :price="parseFloat(result.price)"
```

Comment:
@Zelig880 I don't love that I converted this to a float here. From what I can see, we are saving the price as a `Number`, so I'm not sure why it's coming back as `String` from the DB. Any ideas or thoughts on this?

Reply:
I will have to investigate this. It is probably a miss configuration of the db schema or the way the data is retrieved from the PHP side. Nothing to worry about for now, but good to note